### PR TITLE
docs cleanup

### PR DESCRIPTION
**Pull request by zramsay**
prepare for RTD deprecation and move to tendermint.com/docs

### CODE REVIEW

File: docs/specification.rst [hunk 1]
```diff
@@ -1,21 +0,0 @@
-#############
-Specification
```
Comment (zramsay) [inline docs/specification.rst]: #1814

**Comment by zramsay** [comment]
this is ready

File: docs/README.md [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
-**WARNING:** This documentation is intended to be viewed at:
+Tendermint Core is Byzantine Fault Tolerant (BFT) middleware that takes a state transition machine - written in any programming language - and securely replicates it on many machines. In other words, a blockchain.
 
-https://tendermint.readthedocs.io
+Tendermint requires an application running over the Application Blockchain Interface (ABCI) - and comes packaged with an example application to do so. Follow the [installation instructions](./install) to get up and running quickly. For more details on [using tendermint](./using-tendermint) see that and the following sections.
 
-and may contain broken internal links when viewed from Github.
+## Networks
+
+Testnets can be setup manually on one or more machines, or automatically on one or more machine, using a variety of methods described in the [deploy testnets section](./deploy-testnets). For more information (and to join) about the Cosmos Network testnets, see [here](link).
```
Comment (melekes) [inline docs/README.md]: `see [here](link)` link is missing

Comment (zramsay) [reply docs/README.md]: yeah it's a placeholder. ping @jolesbi or @nylira

Comment (jolesbi) [reply docs/README.md]: `/getting-started/full-node.md`

**Pull request merged**